cifer/chain.json

```diff
@@ -54,8 +54,8 @@
     "persistent_peers": []
   },
   "apis": {
-    "rpc": [],
-    "rest": []
+    "rpc": "https://cif_node.cifer.ai/",
```

Comment:
Please keep the format for rpc and rest. Refer to https://github.com/cosmos/chain-registry/blob/master/cosmoshub/chain.json for an example of how to properly add your RPC and REST end points.

Also please change your PR to an appropriate title.